crates/gosub_styling/tools/generate_definitions/main.go

```diff
@@ -41,7 +42,30 @@ func main() {
 		len(webrefData.Properties), len(webrefData.Values), len(webrefData.AtRules), len(webrefData.Selectors),
 	)
 
+	sort.Slice(webrefData.Properties, func(i, j int) bool {
+		return webrefData.Properties[i].Name < webrefData.Properties[j].Name
+	})
+
```

Comment:
I'm not sure if we want to sort the properties alphabetically, it might make more sense to sort them after its spec.

Reply:
how do you mean?

Reply:
When you are looking for a specific prop, and it's syntax, the computed props might be nearer then, since they are all defined in the same spec.

I guess for expanded props it isn't really an issue, since of `grid` or `mask`, they all also start with `grid-` or `mask`.